Symfony/CS/Fixer/PSR2/ElseifFixer.php

```diff
@@ -57,8 +75,13 @@ public function fix(\SplFileInfo $file, Tokens $tokens)
         if (defined('HHVM_VERSION')) {
             foreach ($tokens->findGivenKind(T_ELSEIF) as $token) {
                 $token->setContent('elseif');
+                $changed = true;
```

Comment:
not needed here. The ``T_ELSEIF`` token type is already registered as found when reaching this :)